ci/.travis.yml

```diff
@@ -12,29 +12,14 @@ before_install:
 bundler_args: "--binstubs --standalone --without documentation --path ../bundle"
 script: "script/run_build"
 rvm:
-  - 1.8.7
-  - 1.9.2
-  - 1.9.3
-  - 2.0.0
-  - ree
-  - rbx-3
   - jruby-9.1.7.0 # pin JRuby to this until travis/rvm can install later versions
   - jruby-head
-  - jruby-1.7
 env:
   - JRUBY_OPTS='--dev'
+  - RSPEC_CI='true'
```

Comment:
You don't need this, its only used by Github to recognise windows CI.

Reply:
As far as I remember, `rspec-rails` sub-build breaks with https://github.com/rspec/rspec-dev/pull/259#discussion_r534432772

I'd rather keep this line since probably we'll get rid of this file altogether soon anyway.